nucliadb_vectors/src/data_types/mod.rs

```diff
@@ -18,19 +18,38 @@
 // along with this program. If not, see <http://www.gnu.org/licenses/>.
 //
 
-pub mod dtrie;
-pub mod merger;
+pub mod dtrie_ram;
+pub mod key_value;
 pub mod trie;
+pub mod trie_ram;
+pub mod vector;
+
+pub mod usize_utils {
+    pub const USIZE_LEN: usize = (usize::BITS / 8) as usize;
+    pub fn usize_from_slice_le(v: &[u8]) -> usize {
+        let mut buff = [0; USIZE_LEN];
+        buff.copy_from_slice(v);
+        usize::from_le_bytes(buff)
+    }
+}
```

Comment:
I suggest to put this nested module into its own file, and maybe name it `num`

Reply:
Mmm.. I dont know about that, why `num`? Is a module used only  for some `usize` specific utilities. With regard to moving it to its own file, would it really be a benefit?

Reply:
We could make advantage of that free-function on integer to start our own 'num' extension in 'nucliadb_core'.
And having it in a separated file let's use isolate the code in a proper way and add tests+documentation without polluting the not related codes.